config/routes.rb

```diff
@@ -2,6 +2,10 @@
   devise_for :users
   root to: 'homepage#show'
   resources :papers
-  resources :lists
+  resources :lists do
+    member do
+      resources :lists_papers, only: [:create, :destroy] 
```

Comment:
The same comment I had on my own PR about treating this like an API counts here, too. We will need to clean this API up around the time we switch to a real frontend.